### PR TITLE
Todays Card View

### PR DESCRIPTION
**Pull request by t1chita**

### CODE REVIEW

File: HW37-Tsvimify-Collaboration/CustomViews/TodaysRow.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import SwiftUI
+
+struct TodaysRow: View {
+    let hourlyForecast: HourlyForecast
+    
+    var body: some View {
+        VStack {
+            Text(String(Int(hourlyForecast.temp)) + "°C")
+                .font(.system(size: 18,weight: .regular))
```
Comment (Tornike222) [inline HW37-Tsvimify-Collaboration/CustomViews/TodaysRow.swift]: Make separated computed properties for these texts there

File: HW37-Tsvimify-Collaboration/CustomViews/WeatherImageView.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+//
+
+import SwiftUI
+
+struct WeatherImageView: View {
+    @StateObject var imageLoader: ImageLoader
+    let imageSize: CGFloat
+    
+    init(url: URL, imageSize: CGFloat) {
+        self._imageLoader = StateObject(wrappedValue: ImageLoader(url: url))
```
Comment (Tornike222) [inline HW37-Tsvimify-Collaboration/CustomViews/WeatherImageView.swift:15]: Why do u need init there? make imageLoader optional and u wont have to write init

File: HW37-Tsvimify-Collaboration/Extensions/DoubleExtensions.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+//
+//  Created by Temur Chitashvili on 13.06.24.
+//
+
+import Foundation
+
+extension Double {
+    func getTimeStringFromUTC() -> String {
+        let date = Date(timeIntervalSince1970: self)
+
```
Comment (Tornike222) [inline HW37-Tsvimify-Collaboration/Extensions/DoubleExtensions.swift]: check extra spaces there and between functions

**Review by Tornike222 — CHANGES_REQUESTED** [review]
wrote some comments, fix and push again

**Review by Gatchu69 — APPROVED** [review]

**Review by Tornike222 — APPROVED** [review]
Everyhting is perfect

**Comment by t1chita** [comment]
okay

**Review by irinkadat — APPROVED** [review]
works fine, looks fine

**Pull request merged**